challenge-05/challenge-05.js

```diff
@@ -47,30 +51,48 @@ propriedades:
 - Se o parâmetro não for passado, a função deve retornar o objeto com todos
 os livros.
 */
-// ?
+function book(nameBook) {
+  var books = {
+    'Guia Front-End': {
+      quantidadePaginas: 174,
+      autor: 'Diego Eis',
+      editora: 'Casa do Código' },
```

Comment:
-1
Para melhorar a leitura, quebre o fechamento do objeto para ficar no mesmo nível de indentação do início do objeto. Deve ficar assim:

```js
'Guia Front-End' : {
  ...
}
```